### ISSUE TITLE
Rescanning Library Does Not Omit Removed Files

### ISSUE THREAD

**Issue by foglabs** [martpie/museeks#742]
Hey! Love the project so far, thanks!

On osx 13.4, I did the following:
-Scanned a folder of music ( Mooz/ )
-Deleted an album from a subfolder of the scanned folder ( Mooz/Lucky Daye/Candydrip )
-Rescanned the folder of music ( Mooz/ )

The Mooz/ folder is on my local machine, within my home folder.

I expected the deleted album to be removed from my library, but it instead remained present. I can retest or provide additional details if needed!

**Comment by martpie** [comment]
Yeah, it's a known limitation, because the other way around would be a bit painful for users to revert. I have a few ideas how we could make it work thought, via storing blacklisted paths in the config, and allowing clearing the blocklist from the UI.

**Comment by martpie** [comment]
Sorry I realize I wasn't super clear. Basically when tracks are removed from the library, they are just removed from the DB, but Museeks does not keep track of those deletion. Meaning during a re-scan, Museeks just re-import them.

The  reason I did not implement a fix for that so far, was that in case someone really want to add them back (after deleting them), they should be able to "delete the deletion" from  the UI, and I did not really find a good way to do that.

But I have a couple of ideas in mind now :)
